src/rebar_prv_path.erl

```diff
@@ -77,15 +77,26 @@ paths([{rel, true}|Rest], Apps, State, Acc) ->
 
 base_dir(State) -> io_lib:format("~s", [rebar_dir:base_dir(State)]).
 bin_dir(State)  -> io_lib:format("~s/bin", [rebar_dir:base_dir(State)]).
-lib_dir(State)  -> io_lib:format("~s/lib", [rebar_dir:base_dir(State)]).
+lib_dir(State)  -> io_lib:format("~s/~s", [rebar_dir:base_dir(State),
+                                           rebar_state:get(State, deps_dir,
+                                                           ?DEFAULT_DEPS_DIR)]).
```

Comment:
Rather than using this format, consider calling `rebar_dir:deps_dir/1` for all of this patch. It seems that using `rebar_dir:base_dir/1` was just an oversight.